pkg/storage/tsdb/block/block_test.go

```diff
@@ -206,7 +213,7 @@ func TestUpload(t *testing.T) {
 		require.Equal(t, 3, len(bkt.Objects()))
 		chunkFileSize := getFileSize(t, filepath.Join(tmpDir, b1.String(), ChunksDirname, "000001"))
 		require.Equal(t, chunkFileSize, int64(len(bkt.Objects()[path.Join(b1.String(), ChunksDirname, "000001")])))
-		require.Equal(t, 401, len(bkt.Objects()[path.Join(b1.String(), IndexFilename)]))
+		require.Equal(t, 418, len(bkt.Objects()[path.Join(b1.String(), IndexFilename)]))
```

Comment:
I wonder if instead we should instead just compare it with the size of the file that was written to disk. Asserting on the number of bytes in an index when testing bucket upload functionality seems flaky